lib/sp_api_helpers.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Module to help create request body for API endpoints
+module SpApiHelpers
+  require "securerandom"
+
+  def self.fees_estimate_request_body(marketplace_id, price, fees_currency_code,
+                                      shipping = nil, points = nil, identifier = SecureRandom.uuid,
+                                      is_amazon_fulfilled = true, optional_fulfillment_program = "FBA_CORE")
+    GetMyFeesEstimateRequest.new(marketplace_id, price, fees_currency_code,
+                                 shipping, points, identifier,
+                                 is_amazon_fulfilled, optional_fulfillment_program)
+  end
+  class GetMyFeesEstimateRequest
```

Comment:
I like how you're using these classes to get the request body, but could we put each class in it's own file? Maybe we could have an `sp_api_helpers` folder within `lib` and these could live there. The reason I say that is because I could foresee this `sp_api_helpers.rb` file growing really quickly if we reuse this code pattern, in which case it would get messy.